src/eredis_cluster.erl

```diff
@@ -195,7 +195,7 @@ handle_transaction_result(Result, Version) ->
         % mapping.
         {error, no_connection} ->
             logger:error("eredis_cluster:handle_transaction_result <--> {error, no_connection}",[]),
-            eredis_cluster_monitor:refresh_mapping(Version),
+            %%eredis_cluster_monitor:refresh_mapping(Version),
```

Comment:
Why did you ingore it？

Reply:
Just testing, uncommented in the last commit e765f1e51b2165e0daa41628f631034dce6178dd